static/js/chart/draw.ts

```diff
@@ -57,6 +111,15 @@ const AXIS_GRID_FILL = "#999";
 // Max Y value used for y domains for charts that have only 0 values.
 const MAX_Y_FOR_ZERO_CHARTS = 10;
 
+function translate(text: string): string {
```

Comment:
nit: name the function "translated"

Reply:
I see, to reduce conflict with d3. Done.